Containerfile-dev

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/389ds/devcontainer
+ARG CUSTOM_CRT_URL
+
+ARG CUSTOM_CRT_URL
+
+RUN if [ -z "$CUSTOM_CRT_URL" ] ; then echo "No custom cert needed"; else \
+          curl -o /etc/pki/ca-trust/source/anchors/customcert.crt $CUSTOM_CRT_URL \
+          && update-ca-trust \
+          ; fi \
+          && dnf install -y openssl nss-tools libasan procps
```

Comment:
I've updated devcontainer image, it now contains these packages.

Reply:
Great.  Consider setting the environment variable default values in the upstream Containerfile as well.  Plus consider setting the entrypoint to use tini with container-entrypoint.sh that handles site-specific cert handling.